nominees/telegram-foss.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Telegram-FOSS",
+  "aliases": [
+    ""
+  ],
+  "description": "A telegram fork that uses FOSS libraries, and with a focus on privacy and deletability.",
+  "website": "https://github.com/Forkgram/TelegramAndroid",
```

Comment:
Not sure how this website relates to the project - could we ask for clarification? Is there another website - otherwise, we could use the Telegram-FOSS repo.